### DISCUSSION TITLE
Unlist 2202.8.2 from nuget.org

### DISCUSSION THREAD

**Discussion by jnyrup** [QuestPDF/QuestPDF#411] [General]
I'll suggest unlisting 2202.8.2 on nuget.org to have the newest 2022.11.0 listed as the newest version.

![image](https://user-images.githubusercontent.com/919634/201317992-61a1e82d-e2df-441d-91a2-adf44ab0d934.png)

2202.8.2 will still be downloadable for users explicitly specifying it.

**Comment by MarcinZiabek** [comment]
Hi 😁 I have unlisted if a couple of weeks ago:

![image](https://user-images.githubusercontent.com/9263853/201323690-4a560ba6-ce12-4029-8054-d6a187893947.png)

![image](https://user-images.githubusercontent.com/9263853/201323817-a2001e70-a1a9-40dc-8653-1409f1de2627.png)

![image](https://user-images.githubusercontent.com/9263853/201323840-efc40d47-2b07-41d0-a009-a042dc9a4d22.png)

Are you aware of any other actions I should perform? Is it possible that you Visual Studio configuration shows also unlisted packages?

**Comment by jnyrup** [comment]
It was because we had it in our cached nuget feed 🤦‍♂️ 
I just had to delete it from there.

FYI, one of my co-workers started using QuestPDF recently and it was a great experience ❤️

**Reply by MarcinZiabek** [reply]
No worries, I'm glad that everything is all right 😁 Thank you for sharing that feedback! I am excited for the next year and all changes planned.

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
